storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,133 @@
+
+Install the component
+---------------------
+
+To start using the page tracking functionality, do the following steps:
+
+1. Install the mysqlbackup component and enable it on the server: 
+
+  .. code-block:: mysql
+
+     $ INSTALL COMPONENT "file://component_mysqlbackup";
+
+2. Check whether the mysqlbackup component is installed succesfuly:
+
+  .. code-block:: bash
+
+     SELECT COUNT(1) FROM mysql.component WHERE component_urn='file://component_mysqlbackup';
+
+Using page tracking
+-------------------
+
+If the server page flushing rate is low or it uses a big buffer pool, you can start page tracking before creating the first backup. Thus, you ensure that page tracking log sequence number (LSN) is more than the checkpoint LSN of the server.
+
+.. note::
+
+   To use the page tracking options, you must have the `Backup_ADMIN <https://docs.oracle.com/cd/E17952_01/mysql-enterprise-backup-8.0-en/mysqlbackup.privileges.html>`__ privileges.
+
+Start page tracking
+-------------------
+
+After the mysqlbackup component is loaded and active on the server, start the page tracking:
+
+  .. code-block:: mysql
+
+     $ SELECT mysqlbackup_page_track_set(true);
+
+.. note::
+
+   PXB processes all the tracked pages in memory. If PXB does not have enough available memory to process all the tracked pages, PXB throws an error and exits. For example, if an incremental backup uses 200GB, PXB can additionally use about 100MB of memory to store the page tracking data.
+
+Enable page tracking
+--------------------
+
+Enable the page tracking functionality for the full and incremental backups with the ``--page-tracking`` option.
+
+The option has the following benefits:
+
+* Resets page tracking to the start of the backup. This reset allows the next incremental backup to use page tracking.
+
+* Allows the use of page tracking for an incremental backup if the page tracking data is available from the backup's start checkpoint LSN.
+
+An example of creating a full backup using the ``--page-tracking`` option.
+
+  .. code-block:: bash
+
+     xtrabackup --backup --target-dir=$FULL_BACK --page-tracking
+
+An example of creating an incremental backup using the ``--page-tracking`` option.
+  
+  .. code-block:: bash
+
+     xtrabackup --backup --target-dir=$INC_BACKUP  --incremental-basedir=$FULL_BACKUP --page-tracking
+
+After enabling the functionality, the next incremental backup finds changed pages using page tracking.
+
+.. note::
+
+   For the very first full backup using page tracking, PXB may have a delay. The following is an example of the message you can receive: 
+
+   .. code-block:: bash
+
+     xtrabackup: pagetracking: Sleeping for 1 second, waiting for checkpoint lsn 17852922 / 
+     to reach to page tracking start lsn 21353759
+   
+   This delay ensures that the page tracking start LSN is more than the backup end checkpoint LSN which is required for the next incremental backup.
```

Comment:
Something like
```suggestion
   This delay ensures that the page tracking start LSN is more than the backup end checkpoint LSN which is required for the next incremental backup.
  To avoid this delay, you can start page tracking before creating the first backup. Thus, you ensure that page tracking log sequence number (LSN) is more than the checkpoint LSN of the server.
  
 Manually start page tracking
  After the mysqlbackup component is loaded and active on the server, start the page tracking:

   .. code-block:: mysql

      $ SELECT mysqlbackup_page_track_set(true);
```

Reply:
updated